driver-core/src/main/com/mongodb/client/model/SimplePipelineStage.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model;
+
+import org.bson.BsonDocument;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.conversions.Bson;
+
+class SimplePipelineStage implements Bson {
```

Comment:
I'd be more inclined to revert this change and just add

>     <suppress checks="FileLength" files="Aggregates"/>

to `config/checkstyle/suppressions.xml`.  Just because it's hard to justify why this one should move but not all the others.

Reply:
Updated